### PR TITLE
i18n(pt-BR): Translate tutorial `pages-2` second half

### PR DESCRIPTION
**Pull request by Yan-Thomas**
#### What kind of changes does this PR include?

- Translated content

#### Description

PT-BR translation for the second half of `pages-2`.

### CODE REVIEW

File: src/content/docs/pt-br/tutorial/2-pages/5.mdx [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+Sua página Sobre agora está estilizada utilizando *ambos* o arquivo `global.css` *e* a tag `<style>`.
+
+- Os estilos de ambos os métodos de estilização estão sendo aplicados? 
+
+    <p>
+      <Spoiler>Sim</Spoiler>
+    </p>
+
+- Há quaisquer estilos conflitantes, e se sim, quais são aplicados
```
Comment (lumamontes) [inline src/content/docs/pt-br/tutorial/2-pages/5.mdx:118]:
```suggestion
- Há quaisquer estilos conflitantes, e se sim, quais são aplicados?
```

File: src/content/docs/pt-br/tutorial/2-pages/5.mdx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+    <p>
+      <Spoiler>Sim, `<h1>` tem um tamanho de `2.5rem` globalmente, porém é `4rem` localmente na tag `<style>`. A regra local `4rem` é aplicada na página Sobre.</Spoiler>
+    </p>
+
+- Descreva como `global.css` e `<style>` trabalham juntos.
+
+    <p>
+      <Spoiler>Quando estilos conflitantes são definidos ambos globalmente e na tag `<style>` local de uma página, os estilos locais devem sobrescrever quaisquer estilos globais. Porém, pode ter outros fatores envolvidos, então sempre inspecione visualmente o seu site para certificar-se que seus estilos estão sendo apropriadamente aplicados!)</Spoiler>
```
Comment (lumamontes) [inline src/content/docs/pt-br/tutorial/2-pages/5.mdx:127]:
```suggestion
      <Spoiler>Quando estilos conflitantes são definidos ambos globalmente e na tag `<style>` local de uma página, os estilos locais devem sobrescrever quaisquer estilos globais. (Porém, pode ter outros fatores envolvidos, então sempre inspecione visualmente o seu site para certificar-se que seus estilos estão sendo apropriadamente aplicados!)</Spoiler>
```

**Review by lumamontes — APPROVED** [review]
i added some small suggestions, but great job! ✨

**Comment by Yan-Thomas** [comment]
Thanks for the great review @lumamontes 💜

**Pull request merged**